app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,19 @@
+body {
+padding: 10%;
+width: 60em;
+margin: 0 auto 0 auto;
```

Comment:
I think what you're trying to get at is that you want me to just use bootstrap to center the board. Right?